To: vim_dev@googlegroups.com
Subject: Patch 7.3.047
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.047 (after 7.3.032)
Problem:    Missing makefile updates for test 75.
Solution:   Update the makefiles.
Files:	    src/testdir/Make_amiga.mak, src/testdir/Make_dos.mak,
	    src/testdir/Makefile, src/testdir/Make_ming.mak,
	    src/testdir/Make_os2.mak, src/testdir/Make_vms.mms


*** ../vim-7.3.046/src/testdir/Make_amiga.mak	2010-10-20 17:44:01.000000000 +0200
--- src/testdir/Make_amiga.mak	2010-10-20 20:43:25.000000000 +0200
***************
*** 27,33 ****
  		test56.out test57.out test58.out test59.out test60.out \
  		test61.out test62.out test63.out test64.out test65.out \
  		test66.out test67.out test68.out test69.out test70.out \
! 		test71.out test72.out test73.out test74.out
  
  .SUFFIXES: .in .out
  
--- 27,33 ----
  		test56.out test57.out test58.out test59.out test60.out \
  		test61.out test62.out test63.out test64.out test65.out \
  		test66.out test67.out test68.out test69.out test70.out \
! 		test71.out test72.out test73.out test74.out test75.out
  
  .SUFFIXES: .in .out
  
***************
*** 121,123 ****
--- 121,124 ----
  test72.out: test72.in
  test73.out: test73.in
  test74.out: test74.in
+ test75.out: test75.in
*** ../vim-7.3.046/src/testdir/Make_dos.mak	2010-10-20 17:44:01.000000000 +0200
--- src/testdir/Make_dos.mak	2010-10-20 20:43:48.000000000 +0200
***************
*** 28,34 ****
  		test37.out test38.out test39.out test40.out test41.out \
  		test42.out test52.out test65.out test66.out test67.out \
  		test68.out test69.out test71.out test72.out test73.out \
! 		test74.out
  
  SCRIPTS32 =	test50.out test70.out
  
--- 28,34 ----
  		test37.out test38.out test39.out test40.out test41.out \
  		test42.out test52.out test65.out test66.out test67.out \
  		test68.out test69.out test71.out test72.out test73.out \
! 		test74.out test75.out
  
  SCRIPTS32 =	test50.out test70.out
  
*** ../vim-7.3.046/src/testdir/Makefile	2010-10-20 17:44:01.000000000 +0200
--- src/testdir/Makefile	2010-10-20 20:44:10.000000000 +0200
***************
*** 25,31 ****
  		test59.out test60.out test61.out test62.out test63.out \
  		test64.out test65.out test66.out test67.out test68.out \
  		test69.out test70.out test71.out test72.out test73.out \
! 		test74.out
  
  SCRIPTS_GUI = test16.out
  
--- 25,31 ----
  		test59.out test60.out test61.out test62.out test63.out \
  		test64.out test65.out test66.out test67.out test68.out \
  		test69.out test70.out test71.out test72.out test73.out \
! 		test74.out test75.out
  
  SCRIPTS_GUI = test16.out
  
*** ../vim-7.3.046/src/testdir/Make_ming.mak	2010-10-20 17:44:01.000000000 +0200
--- src/testdir/Make_ming.mak	2010-10-20 20:43:53.000000000 +0200
***************
*** 48,54 ****
  		test37.out test38.out test39.out test40.out test41.out \
  		test42.out test52.out test65.out test66.out test67.out \
  		test68.out test69.out test71.out test72.out test73.out \
! 		test74.out
  
  SCRIPTS32 =	test50.out test70.out
  
--- 48,54 ----
  		test37.out test38.out test39.out test40.out test41.out \
  		test42.out test52.out test65.out test66.out test67.out \
  		test68.out test69.out test71.out test72.out test73.out \
! 		test74.out test75.out
  
  SCRIPTS32 =	test50.out test70.out
  
*** ../vim-7.3.046/src/testdir/Make_os2.mak	2010-10-20 17:44:01.000000000 +0200
--- src/testdir/Make_os2.mak	2010-10-20 20:44:00.000000000 +0200
***************
*** 27,33 ****
  		test56.out test57.out test58.out test59.out test60.out \
  		test61.out test62.out test63.out test64.out test65.out \
  		test66.out test67.out test68.out test69.out test70.out \
! 		test71.out test72.out test73.out test74.out
  
  .SUFFIXES: .in .out
  
--- 27,33 ----
  		test56.out test57.out test58.out test59.out test60.out \
  		test61.out test62.out test63.out test64.out test65.out \
  		test66.out test67.out test68.out test69.out test70.out \
! 		test71.out test72.out test73.out test74.out test75.out
  
  .SUFFIXES: .in .out
  
*** ../vim-7.3.046/src/testdir/Make_vms.mms	2010-10-20 17:44:01.000000000 +0200
--- src/testdir/Make_vms.mms	2010-10-20 20:44:05.000000000 +0200
***************
*** 4,10 ****
  # Authors:	Zoltan Arpadffy, <arpadffy@polarhome.com>
  #		Sandor Kopanyi,  <sandor.kopanyi@mailbox.hu>
  #
! # Last change:  2010 Aug 04
  #
  # This has been tested on VMS 6.2 to 8.3 on DEC Alpha, VAX and IA64.
  # Edit the lines in the Configuration section below to select.
--- 4,10 ----
  # Authors:	Zoltan Arpadffy, <arpadffy@polarhome.com>
  #		Sandor Kopanyi,  <sandor.kopanyi@mailbox.hu>
  #
! # Last change:  2010 Oct 20
  #
  # This has been tested on VMS 6.2 to 8.3 on DEC Alpha, VAX and IA64.
  # Edit the lines in the Configuration section below to select.
***************
*** 74,80 ****
  	 test56.out test57.out test60.out \
  	 test61.out test62.out test63.out test64.out test65.out \
  	 test66.out test67.out test68.out test69.out \
! 	 test71.out test72.out test74.out
  
  # Known problems:
  # Test 30: a problem around mac format - unknown reason
--- 74,80 ----
  	 test56.out test57.out test60.out \
  	 test61.out test62.out test63.out test64.out test65.out \
  	 test66.out test67.out test68.out test69.out \
! 	 test71.out test72.out test74.out test75.out
  
  # Known problems:
  # Test 30: a problem around mac format - unknown reason
*** ../vim-7.3.046/src/version.c	2010-10-27 17:40:53.000000000 +0200
--- src/version.c	2010-10-27 18:34:20.000000000 +0200
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     47,
  /**/

-- 
5 out of 4 people have trouble with fractions.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\        download, build and distribute -- http://www.A-A-P.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
